selfdrive/car/honda/values.py

```diff
@@ -335,21 +343,17 @@ class CAR:
   CAR.ACCORDH: {
     (Ecu.gateway, 0x18daeff1, None): [
       b'38897-TWA-A120\x00\x00',
-      b'38897-TWD-J020\x00\x00',
```

Comment:
why are these removed?

Reply:
This commit fixes an ID conflict. These were added with https://github.com/commaai/openpilot/pull/20850 and they match a possible 2021 Hybrid. Some of the deleted IDs could be from the older models, but I have no way of knowing which cars they're really from. @pd0wm are you able to see if these came from a car that doesn't have `36161-TWA-A330\x00\x00` or `36161-TVC-A330\x00\x00`?

Removing at least the camera and radar IDs (definitely only for 2021+) could fix the conflict.

Reply:
I've rebased and limited commit #2 to just the camera and radar firmware ids for the 2021 Hybrid. I'm happy to submit the PR to clean up the rest later once you all are ready for that.